crates/bevy_landmass/src/debug.rs

```diff
@@ -321,39 +415,52 @@ impl<CS: CoordinateSystem> DebugDrawer<CS> for GizmoDrawer<'_, '_, '_, CS> {
             0.01,
             line[0].distance(line[1]),
           )),
-        Color::srgba(0.0, 1.0, 0.0, 0.6),
+        color,
       );
       return;
     }
-    self.0.line(
+    let Some(color) = (match line_type {
+      LineType::BoundaryEdge => self.config_ext.boundary_edge,
+      LineType::ConnectivityEdge => self.config_ext.connectivity_edge,
+      LineType::HeightEdge => self.config_ext.height_edge,
+      LineType::BoundaryLink => self.config_ext.boundary_link,
+      LineType::AgentCorridor(..) => self.config_ext.agent_corridor,
+      LineType::Target(..) => self.config_ext.target_line,
+      LineType::Waypoint(..) => self.config_ext.waypoint_line,
+    }) else {
+      return;
+    };
+    self.line(
       CS::to_world_position(&line[0]),
       CS::to_world_position(&line[1]),
-      match line_type {
-        LineType::BoundaryEdge => Color::srgba(0.0, 0.0, 1.0, 0.6),
-        LineType::ConnectivityEdge => Color::srgba(0.5, 0.5, 1.0, 0.6),
-        LineType::HeightEdge => Color::srgba_u8(33, 102, 57, 128),
-        LineType::BoundaryLink => unreachable!(),
-        LineType::AgentCorridor(_) => Color::srgba(0.6, 0.0, 0.6, 0.6),
-        LineType::Target(_) => Color::srgba(1.0, 1.0, 0.0, 0.6),
-        LineType::Waypoint(_) => Color::srgba(0.6, 0.6, 0.6, 0.6),
-      },
+      color,
     );
   }
 
   fn add_triangle(
     &mut self,
-    _triangle_type: TriangleType,
-    _triangle: [CS::Coordinate; 3],
+    triangle_type: TriangleType,
+    triangle: [CS::Coordinate; 3],
   ) {
-    // Bevy doesn't have a way to draw triangles :'(
+    let Some(color) = (match triangle_type {
```

Comment:
I'm not sure there's any point to this? The edges of nodes are already drawn. This will just duplicate those edges in a different colour (probably causing z-fighting)?

Reply:
Ohhh I guess you would see the interiors of nodes? Not sure that's valuable though since it's a pretty dumb triangulation...